runtime/compiler/env/j9methodServer.cpp

```diff
@@ -1644,6 +1641,7 @@ TR_ResolvedJ9JITaaSServerMethod::addValidationRecordForCachedResolvedMethod(cons
 bool
 TR_ResolvedJ9JITaaSServerMethod::validateMethodFieldAttributes(const TR_J9MethodFieldAttributes &attributes, bool isStatic, int32_t cpIndex, bool isStore, bool needAOTValidation)
    {
+   if (attributes.isUnresolvedInCP()) return true; // Validation should not be done against attributes cached per-resolved method
```

Comment:
Please put `return true` on the next lie to adhere to our coding conventions.

Reply:
I just made that change.